stubs/ORM/Tools/Pagination/Paginator.stub

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Doctrine\ORM\Tools\Pagination;
+
+use ArrayIterator;
+use Countable;
+use Doctrine\ORM\Query;
+use Doctrine\ORM\QueryBuilder;
+use IteratorAggregate;
+
+/**
+ * @template T
```

Comment:
unlike the [original](https://github.com/doctrine/orm/blob/2.16.x/lib/Doctrine/ORM/Tools/Pagination/Paginator.php#L30) this was changed to invariant because otherwise the return from `getIterator()` in l27 would throw a error